### PR TITLE
passing output_filename to fuel_convert

### PR DESCRIPTION
**Pull request by sygi**
Fixes just created #334.

### CODE REVIEW

**Comment by rizar** [comment]
This looks good to me, but I am not knowledgeable enough in this part of Fuel to merge it myself.

@vdumoulin , @bartvm , here is a very small PR, can you please take a look?

File: fuel/bin/fuel_convert.py [hunk 1]
```diff
@@ -58,12 +58,17 @@ def main(args=None):
...
+        subparser.add_argument(
+            "-r", "--output_filename", help="new name of the created dataset",
+            type=str, default=None)
         # Allows the parser to know which subparser was called.
         subparser.set_defaults(which_=name)
         convert_functions[name] = fill_subparser(subparser)
 
     args = parser.parse_args(args)
     args_dict = vars(args)
+    if args_dict['output_filename'] is not None:
```
Comment (bartvm) [inline fuel/bin/fuel_convert.py]: Maybe add something like `and os.path.splitext(args_dict['output_filename'])[1] not in ('.hdf5', '.hdf', '.h5')`?

**Comment by bartvm** [comment]
LGTM to me as well, maybe have a look at the one comment I made.

File: fuel/bin/fuel_convert.py (hunk 1 above)
Comment (sygi) [inline fuel/bin/fuel_convert.py]: Good idea, thanks. Done.

**Comment by rizar** [comment]
In fact, that PR has broken a lot of converters. Namely, all those in which `output_filename` was a keyword argument with a default value. Including https://github.com/mila-udem/fuel/blob/master/fuel/converters/dogs_vs_cats.py#L17

**Pull request merged**